website/source/guides/identity/secure-intro.html.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: "guides"
+page_title: "Secure Introduction of Vault Clients - Guides"
+sidebar_current: "guides-identity-secure-intro"
+description: |-
+  This introductory guide walk through the mechanism of Vault clients to
+  authenticate with Vault. There are two approaches at a high-level: platform
+  integration, and trusted orchestrator.
+---
+
+# Secure Introduction of Vault Clients
+
+A _secret_ is something that will elevate the risk if exposed to unauthorized
+entities and results in undesired consequences (e.g. unauthorized data access);
+therefore, only the ***trusted entities*** should have an access to your
+secrets.
+
+If you can securely get the first secret from an originator to a consumer,
+all subsequent secrets transmitted between this originator and consumer can be
+authenticated with the trust established by the successful distribution and user
+of that first secret.  
```

Comment:
Getting the first secret to the consumer, is the _secure introduction_ challenge.